src/services/PinataIpfsClient.ts

```diff
@@ -32,7 +32,7 @@ export class PinataIpfsClient implements IIpfsClient {
 
   public async addAndPinData(data: string, name?: string): Promise<Hash> {
     const params = new FormData();
-    params.append("file", Buffer.from(data));
+    params.append("file", Buffer.from(data).toString());
```

Comment:
Why?  I don't think this is correct.

Reply:
Sorry about this, this came from a new commit. The issue here was that the app refused to compile because of some error from that function which I fixed quickly with that just to get the app up and running. I had excluded this file but forgot to exclude it in subsequent commits.

I will revert this back